README.md

```diff
@@ -22,6 +22,15 @@ Read the full Parse Server guide here: https://github.com/ParsePlatform/parse-se
 * You now have a database named "dev" that contains your Parse data
 * Install ngrok and you can test with devices
 
+### These scripts can help you to develop your app for Parse Server
```

Comment:
Please keep headings short and longer text in the paragraph. Also check all other headlines.

You will see why when you insert / update the table of contents in the readme.